docs/user-management/account-types.md

```diff
@@ -5,31 +5,36 @@ contentType: reference
 ///
 
-* Owner: this is the account that set up user management. There's one owner account for each n8n instance. You can't transfer ownership to another user, but you can change the email of the owner account instead.
-  The owner can:
-    * Add and remove users, including admin users
-	* Upgrade members to admin, and downgrade admins to member
-    * See and share all workflows    
-	* See, edit, and share all credentials (but not see the sensitive information)
-	* Delete tags
-	* Set up and use [Source control](/source-control-environments/)
-* Admin: elevated permissions within the app.
-  An admin can do everything that an owner can, except:
-	* Access the Cloud dashboard
-	* Modify the owner or change the owner role
-* Members: these are normal n8n users.
-  Members can:
-    * See all workflow tags, create new tags, and assign tags to their workflows. Members can't delete tags.
-    * Change their own password.
-    * Change their own email.
-    * See their own workflows.
+/// note | Account types and role types
+Account types and role types are different things. Role types are part of [RBAC](/user-management/rbac/). 
+
+Every account has one type. The account can have different [role types](/user-management/rbac/role-types/) for different [projects](/user-management/rbac/projects/).
+///
 
 /// note | Create a member-level account for the owner
-We recommend that owners create a member-level account for themselves. Owners can see all workflows, but there is no way to see who created a particular workflow, so there is a risk of overriding other people's work if you build and edit workflows as an owner.
+n8n recommends that owners create a member-level account for themselves. Owners can see all workflows, but there is no way to see who created a particular workflow, so there is a risk of overriding other people's work if you build and edit workflows as an owner.
 ///
+
+
+| Permission | Owner | Admin | Member |
+| ---------- |------ | ----- | ------ |
+| Manage own email and password | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Manage own workflows | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| View, create, and use tags | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Delete tags | :white_check_mark: | :white_check_mark: | :x: |
+| View and share all workflows | :white_check_mark: | :white_check_mark: | :x: |
+| View, edit, and share all credentials | :white_check_mark: | :white_check_mark: | :x: |
+| Set up and use [Source control](/source-control-environments/) | :white_check_mark: | :white_check_mark: | :x: |
+| Create [projects](/user-management/rbac/projects/) | :white_check_mark: | :white_check_mark: | :x: |
+| View all projects | :white_check_mark: | :white_check_mark: | :x: |
+| Add and remove users | :white_check_mark: | :white_check_mark: | :x: |
+| Modify the owner or change the owner role | :white_check_mark: | :x: | :x: |
```

Comment:
Owner role is immutable, what can ben done is change other user's type between member and admin

Reply:
Is it possible to transfer the owner role to a new person?

Reply:
No, the owner role is fixed and that user cannot even be deleted at this point @StarfallProjects

Reply:
Thank you - fixed.